src/clj/commiteth/eth/core.clj

```diff
@@ -271,3 +270,11 @@
             (filter true?)
             (empty?)))
           true))))
+
+(defn bot-account-eth-threshold-overtaken? []
+  (let [eth (-> (get-balance-wei (eth-account)) wei->eth)]
+    (< eth (env :eth-account-eth-threshold-warning))))
+
+(defn check-bot-account-eth-threshold []
+  (when (bot-account-eth-threshold-overtaken?)
+    (post "NOTIFICATION_URL")))
```

Comment:
* Notification URL can go to config variable `:slack-webhook-url`. If empty, log/error something and abort startup
* No need to make the threshold configurable, just use 0.2 ETH.
* POST contents can be "URGENT: SOB bot account ETH will run out soon, current balance XXX"

Reply:
I will give you the webhook URL privately.